x-pack/docs/en/rest-api/security/get-tokens.asciidoc

```diff
@@ -161,7 +180,26 @@ seconds) that the token expires in, the type, and the refresh token:
   "access_token" : "dGhpcyBpcyBub3QgYSByZWFsIHRva2VuIGJ1dCBpdCBpcyBvbmx5IHRlc3QgZGF0YS4gZG8gbm90IHRyeSB0byByZWFkIHRva2VuIQ==",
   "type" : "Bearer",
   "expires_in" : 1200,
-  "refresh_token": "vLBPvmAB6KvwvJZr27cS"
+  "refresh_token": "vLBPvmAB6KvwvJZr27cS",
+  "authentication" : {
+   "username" : "test_admin",
+   "roles" : [
+    "superuser"
+    ],
+   "full_name" : null,
+   "email" : null,
+   "metadata" : { },
+   "enabled" : true,
+   "authentication_realm" : {
+    "name" : "file",
+    "type" : "file"
+    },
+   "lookup_realm" : {
+    "name" : "file",
+    "type" : "file"
+   },
+   "authentication_type" : "realm"
```

Comment:
The format of the authentication field is inconsistent. Sorry for being picky on this since it is user facing documentation. I think we should strive to ensure:
* The indentation is 2 spaces per level instead of 1. 
* The closing curly and square brackets should align to the corresponding key

This comment applies to all places (some are better formatted than others) where the authentcation field is shown. 

An example of the above format is like the follows (please note that `.` is used in place of whitespace to make it easier to show indentation. ):
```text
{
.."authentication" : {
...."username" : "test_admin",
...."roles" : [
......"superuser"
....],
...."full_name" : null,
...."email" : null,
...."metadata" : {},
...."enabled" : true,
...."authentication_realm" : {
......"name" : "file",
......"type" : "file"
....},
...."lookup_realm" : {
......"name" : "file",
......"type" : "file"
....},
...."authentication_type" : "realm"
..}
}
```

You can also refer to [get-role-mappings.asciidoc](https://raw.githubusercontent.com/elastic/elasticsearch/7.10/x-pack/docs/en/rest-api/security/get-role-mappings.asciidoc)

Reply:
Addressed in #63792